azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -357,4 +357,12 @@ private void assertTwoFlowSame(final ExecutableFlow flow1, final ExecutableFlow
     assertThat(new HashSet<>(flow1.getEndNodes())).isEqualTo(new HashSet<>(flow2.getEndNodes()));
   }
 
+  /**
+   * restores the clock; see {@link #testFetchEmptyRecentlyFinishedFlows()}
+   */
+  @After
+  public void clockReset() {
+    DateTimeUtils.setCurrentMillisOffset(0);
```

Comment:
I checked DateTimeUtils#setCurrentMillisSystem – calling setCurrentMillisOffset(0) is effectively the same. So why not use the simpler method that doesn't require an arg?

Reply:
reuse of existing code; but sure we can replace it as well :)